ExporterStreamProcessorTest#.shouldExecuteScheduledTask is flaky

```
15:02:11 [1 - Java Tests] [ERROR] shouldExecuteScheduledTask(io.zeebe.broker.exporter.stream.ExporterStreamProcessorTest)  Time elapsed: 10.045 s  <<< FAILURE!
15:02:11 [1 - Java Tests] org.junit.ComparisonFailure: expected:<[tru]e> but was:<[fals]e>
15:02:11 [1 - Java Tests] 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
15:02:11 [1 - Java Tests] 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
15:02:11 [1 - Java Tests] 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
15:02:11 [1 - Java Tests] 	at io.zeebe.test.util.TestUtil$Invocation.until(TestUtil.java:96)
15:02:11 [1 - Java Tests] 	at io.zeebe.test.util.TestUtil$Invocation.until(TestUtil.java:69)
15:02:11 [1 - Java Tests] 	at io.zeebe.test.util.TestUtil.waitUntil(TestUtil.java:41)
15:02:11 [1 - Java Tests] 	at io.zeebe.broker.exporter.stream.ExporterStreamProcessorTest.shouldExecuteScheduledTask(ExporterStreamProcessorTest.java:338)
15:02:11 [1 - Java Tests] 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
15:02:11 [1 - Java Tests] 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
15:02:11 [1 - Java Tests] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
15:02:11 [1 - Java Tests] 	at java.lang.reflect.Method.invoke(Method.java:498)
15:02:11 [1 - Java Tests] 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
15:02:11 [1 - Java Tests] 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
15:02:11 [1 - Java Tests] 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
15:02:11 [1 - Java Tests] 	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
15:02:11 [1 - Java Tests] 	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
15:02:11 [1 - Java Tests] 	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
15:02:11 [1 - Java Tests] 	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
15:02:11 [1 - Java Tests] 	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
15:02:11 [1 - Java Tests] 	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
15:02:11 [1 - Java Tests] 	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
15:02:11 [1 - Java Tests] 	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
15:02:11 [1 - Java Tests] 	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
15:02:11 [1 - Java Tests] 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
15:02:11 [1 - Java Tests] 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
15:02:11 [1 - Java Tests] 	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
15:02:11 [1 - Java Tests] 	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
15:02:11 [1 - Java Tests] 	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
15:02:11 [1 - Java Tests] 	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
15:02:11 [1 - Java Tests] 	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
15:02:11 [1 - Java Tests] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
15:02:11 [1 - Java Tests] 	at org.junit.runners.Suite.runChild(Suite.java:128)
15:02:11 [1 - Java Tests] 	at org.junit.runners.Suite.runChild(Suite.java:27)
15:02:11 [1 - Java Tests] 	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
15:02:11 [1 - Java Tests] 	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
15:02:11 [1 - Java Tests] 	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
15:02:11 [1 - Java Tests] 	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
15:02:11 [1 - Java Tests] 	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
15:02:11 [1 - Java Tests] 	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
15:02:11 [1 - Java Tests] 	at org.apache.maven.surefire.junitcore.JUnitCore.run(JUnitCore.java:55)
15:02:11 [1 - Java Tests] 	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.createRequestAndRun(JUnitCoreWrapper.java:137)
15:02:11 [1 - Java Tests] 	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.executeLazy(JUnitCoreWrapper.java:119)
15:02:11 [1 - Java Tests] 	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:87)
15:02:11 [1 - Java Tests] 	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:75)
15:02:11 [1 - Java Tests] 	at org.apache.maven.surefire.junitcore.JUnitCoreProvider.invoke(JUnitCoreProvider.java:158)
15:02:11 [1 - Java Tests] 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
15:02:11 [1 - Java Tests] 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
15:02:11 [1 - Java Tests] 	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
15:02:11 [1 - Java Tests] 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
15:02:11 [1 - Java Tests] 
```

--- a/broker-core/src/test/java/io/zeebe/broker/exporter/stream/ExporterStreamProcessorTest.java
+++ b/broker-core/src/test/java/io/zeebe/broker/exporter/stream/ExporterStreamProcessorTest.java
@@ -324,22 +324,24 @@ public class ExporterStreamProcessorTest {
   public void shouldExecuteScheduledTask() throws Exception {
     // given
     final List<ExporterDescriptor> mockedExporters = createMockedExporters(1);
-    final CountDownLatch latch = new CountDownLatch(1);
+    final CountDownLatch timerTriggerLatch = new CountDownLatch(1);
+    final CountDownLatch timerScheduledLatch = new CountDownLatch(1);
     final Duration delay = Duration.ofSeconds(10);
 
     final ControlledTestExporter controlledTestExporter = exporters.get(0);
     controlledTestExporter.onExport(
-        r -> controlledTestExporter.getController().scheduleTask(delay, latch::countDown));
+        r -> {
+          controlledTestExporter.getController().scheduleTask(delay, timerTriggerLatch::countDown);
+          timerScheduledLatch.countDown();
+        });
 
-    final StreamProcessorControl control =
-        rule.runStreamProcessor((db, dbContext) -> createStreamProcessor(db, mockedExporters));
-    final long position = writeEvent();
-    control.blockAfterEvent(e -> e.getPosition() >= position);
-    TestUtil.waitUntil(control::isBlocked);
+    rule.runStreamProcessor((db, dbContext) -> createStreamProcessor(db, mockedExporters));
+    writeEvent();
+    timerScheduledLatch.await();
 
     // when
     rule.getClock().addTime(delay.plusSeconds(20));
-    latch.await();
+    timerTriggerLatch.await();
 
     // then
   }

